python/mermaid_csv_reader.py

```diff
@@ -35,25 +36,44 @@ def molecules_json(df):
     return molecules_dict
 
 
+def image_json():
```

Comment:
I think of JSON as being a particular serialization, but this returns a dict. Consider either renaming, or returning a JSON string.